test_utils/scripts/update_docs.sh

```diff
@@ -24,7 +24,7 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Function to build the docs.
 function build_docs {
     rm -rf docs/_build/
-    sphinx-build -W -b html -d docs/_build/doctrees docs/ docs/_build/html/
```

Comment:
@lukesneeringer I am :-1: to letting Sphinx warn. Typically these are warnings that we are

- doing something wrong
- using a package which is doing something wrong
- using a feature which will be deprecated

I all cases, we should **see** these warnings, which will never happen if we just let the docs pass on CI.